plugins/catalog-unprocessed-entities/src/components/FailedEntities.tsx

```diff
@@ -93,14 +93,18 @@ const RenderErrorContext = ({
  * easily understand the times.
  */
 const convertTimeToLocalTimezone = (strDateTime: string | Date) => {
-  const dateTime = DateTime.fromFormat(
+  let dateTime = DateTime.fromFormat(
     strDateTime.toLocaleString(),
     'yyyy-MM-dd hh:mm:ss',
     {
       zone: 'UTC',
     },
   );
 
+  if (!dateTime.isValid) {
+    dateTime = DateTime.fromISO(strDateTime.toLocaleString());
```

Comment:
Quick question: Is it possible to use another format besides ISO? If yes, how about adding a configuration to `app-config.yaml` so the user can specify the date format string there?

Reply:
The problem is that timestamps like "2024-11-26T14:57:09.131Z" that includes T and Z cannot be parsed with `toLocaleString`. I tried this but it did not work for me. 

```typescript

'yyyy-MM-ddThh:mm:ss.SSSZ',
    {
      zone: 'UTC',

```

I don't know what things could trigger `FailedEntities` from the first place and what types of time formats to expect. The documentation from Luxton shows that `fromISO` support a wide range of formats. It may be possible to replace the entire `toLocalString()` part with only `fromISO` but I don't know if that could break functionality with something else. See Luxton [documentation](https://moment.github.io/luxon/#/parsing?id=iso-8601)

I assume that, providing the input format in `app-config.yaml` would need some extra parsing implemented in this case.
However, the output format feels more convient to specify in the that file. I could create a separate issue/PR for it if sounds useful.

I'm thinking of this 
https://github.com/backstage/backstage/blob/master/plugins/catalog-unprocessed-entities/src/components/FailedEntities.tsx?plain=1#L106